.github/workflows/update-module.yml

```diff
@@ -0,0 +1,71 @@
+name: Update common modulefiles on Gadi
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+    paths:
+      - 'modules/**'
+jobs:
+  setup-deployment-env:
+    name: Setup Deployment Environment
+    runs-on: ubuntu-latest
+    outputs:
+      deployment-environments: ${{ steps.get-deployment-environment.outputs.deployment-environments }}
+    steps:
+      - name: Checkout config
+        uses: actions/checkout@v4
+
+      - name: Get environments
+        id: get-deployment-environment
+        run: echo "deployment-environments=$(jq --compact-output '.environments' ./config/deployment-environment.json)" >> $GITHUB_OUTPUT
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: setup-deployment-env
+    strategy:
+      fail-fast: false
+      matrix:
+        deployment-environment: ${{ fromJson(needs.setup-deployment-env.outputs.deployment-environments) }}
+    environment: ${{ matrix.deployment-environment }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: access-nri/actions/.github/actions/setup-ssh@main
+        id: ssh
+        with:
+          hosts: |
+            ${{ secrets.HOST_DATA }}
+          private-key: ${{ secrets.SSH_KEY }}
+
+      - name: Generate release modulesfiles
+        run: |
+          mkdir -p release/modules
+          sed 's|{{MODULE_DIR}}|'"${{ vars.MODULE_DIR }}"'|g' modules/.common > release/modules/.common
+          sed 's|{{APPS_DIR}}|'"${{ vars.APPS_DIR }}"'|g' modules/env.sh > release/modules/env.sh
+          chmod +x release/modules/env.sh
+
+    #   - name: Generate pre-release modulesfiles
```

Comment:
I wonder if it's worth removing the pre-release stuff for now?

Reply:
Removed in https://github.com/ACCESS-NRI/access-ram-condaenv/pull/2/commits/de5954f7b4f6665aee5e151f778e6e685fc6c36b